aya/src/maps/mod.rs

```diff
@@ -243,6 +280,15 @@ impl Map {
         let c_name = CString::new(name).map_err(|_| MapError::InvalidName { name: name.into() })?;
 
         let fd = bpf_create_map(&c_name, &self.obj.def).map_err(|(code, io_error)| {
+            let kernel_version = {
```

Comment:
you should compare tuples here like it's done in other places, eg see https://github.com/aya-rs/aya/blob/b2a6f00212997a997799c88ba9022a69d9a0b582/aya/src/programs/cgroup_sock_addr.rs#L77